server/decode.go

```diff
@@ -56,7 +63,16 @@ func readFileDescriptors(files map[string]string) ([]*desc.FileDescriptor, error
 	}
 
 	var p protoparse.Parser
-	p.Accessor = protoparse.FileContentsFromMap(files)
+	p.InferImportPaths = true
+
+	p.Accessor = func(f string) (io.ReadCloser, error) {
```

Comment:
What is the purpose of this Accessor? Can you leave a comment?

Reply:
Added comment